### PR TITLE
Update names in MAGMA restrictions

### PR DESCRIPTION
**Pull request by nbeams**
This PR renames `indices` --> `offsets` in MAGMA restrictions to match the new name in #513 .  Restriction kernel comments have also been updated to reflect these changes.

### CODE REVIEW

File: backends/magma/magma_drestrictApply.cu [hunk 1]
```diff
@@ -158,22 +150,22 @@ magma_readDofsStrided(const magma_int_t NCOMP, const magma_int_t esize,
...
 
     magma_writeDofsOffset_kernel<<<grid, threads, 0, NULL>>>(NCOMP, compstride,
-      esize, nelem, indices, du, dv);
+      esize, nelem, offsets, du, dv);
 }
 
 // WriteDofs from device memory, strided description for L-vector
 // du is E-vector, size nelem * esize * NCOMP
-// dv is L-vector, size nnodes * NCOMP 
+// dv is L-vector, size lsize * NCOMP
```
Comment (jeremylt) [inline backends/magma/magma_drestrictApply.cu:168]: L-vector is size `lsize`

Comment (nbeams) [reply backends/magma/magma_drestrictApply.cu]: Fixed, thanks

**Comment by jeremylt** [comment]
The Travis failure is unrelated - its the intermittent OCCA + OMP + ARM issue

**Comment by jedbrown** [comment]
Merge when ready. @nbeams I think you should have permission.

**Pull request merged**